javascript/mobile.js

```diff
@@ -0,0 +1,31 @@
+const contactList = document.getElementById('contactList');
+const hambugerButton = document.getElementById('hamburgerButton');
+const hambugerButton2 = document.getElementById('hamburgerButton2');
+const options = document.getElementsByClassName('options');
+
+hambugerButton.addEventListener('click', () => {
+  contactList.classList.toggle('appear');
+  document.body.style.overflow = 'hidden';
+});
+
+hambugerButton2.addEventListener('click', () => {
+  contactList.classList.remove('appear');
+  document.body.style.overflow = 'visible';
+});
+
+console.log(options);
```

Comment:
- [ ] Please it's recommended to use JS best practices. So I recommend removing all the console.log in your file.